core/src/main/java/com/novoda/noplayer/internal/exoplayer/NoPlayerAdsLoader.java

```diff
@@ -95,9 +89,7 @@ public void onAdvertsLoaded(List<AdvertBreak> breaks) {
             AdvertPlaybackState advertPlaybackState = AdvertPlaybackState.from(breaks, advertBreakResumePosition);
             advertBreaks = advertPlaybackState.advertBreaks();
             adPlaybackState = advertPlaybackState.adPlaybackState();
-            if (contentInitialPosition > 0) {
```

Comment:
We've had a few chats about this now and if the user triggers playback from a resume position then they are subject to any advert that may be associated with that section of content.